### PR TITLE
Texture-weighted automatic exposure

### PR DESCRIPTION
**Pull request by FrancescoC-unity**
Add an additional metering mode that uses a texture to get the weights to assign to sampled pixels from the source buffer. 

For QA: I verified via Renderdoc to see the shape of the mask in the exposure process 😄 
 But can be verifiable in any project if you change the mask so that it gives more weight to (different) specific areas.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Documentation~/Override-Exposure.md [hunk 1]
```diff
@@ -65,6 +65,10 @@ To configure **Automatic Mode**, select the **Metering Mode**. This tells the Ca
 
 ![](Images/Override-Exposure3.png)
 
+- **Mask Weighted**: The Camera applies a weight to every pixel in the buffer and then uses them to measure the exposure. The weighting is specified by the texture provided in the **Weight Texture Mask** field. Note that if no texture is provided, then this metering mode is equivalent to Average.
```
Comment (JordanL8) [inline com.unity.render-pipelines.high-definition/Documentation~/Override-Exposure.md:68]: **Mask Weighted**: The Camera applies a weight to every pixel in the buffer then uses the weights to measure the exposure. To specify the weighting, this technique uses the Texture set in the **Weight Texture Mask** field. Note that, if you do not provide a Texture, this metering mode is equivalent to **Average**.

**Comment by TomasKiniulis** [comment]
Checked weight texture mask with Automatic and Curve Mapping modes. Works as expected. However, there seems to be an issue where disabled "Weight Texture Mask" override exposes the scene to white, so it's not correctly reverted to Average. This does not occur when override is enabled and does not depend on the mask provided or not.
nU9kbYAS6k

**Comment by FrancescoC-unity** [comment]
> Checked weight texture mask with Automatic and Curve Mapping modes. Works as expected. However, there seems to be an issue where disabled "Weight Texture Mask" override exposes the scene to white, so it's not correctly reverted to Average. This does not occur when override is enabled and does not depend on the mask provided or not.
> nU9kbYAS6k

Nice! Thanks for checking will fix it up

**Comment by FrancescoC-unity** [comment]
> Checked weight texture mask with Automatic and Curve Mapping modes. Works as expected. However, there seems to be an issue where disabled "Weight Texture Mask" override exposes the scene to white, so it's not correctly reverted to Average. This does not occur when override is enabled and does not depend on the mask provided or not.


Fixed

**Pull request merged**